Filters/exclusions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1018
+||stcblink.nypost.com^
```

Comment:
Wait a second, this is for exceptions.
```
@@||stcblink.nypost.com^|
```

Reply:
First you need to figure out why this domain was blocked

Reply:
Moved rule